extensions/librdkafka/ConsumeKafka.cpp

```diff
@@ -185,7 +205,11 @@ void ConsumeKafka::initialize() {
     DuplicateHeaderHandling,
     MaxPollRecords,
     MaxPollTime,
-    SessionTimeout
+    SessionTimeout,
+    SecurityCA,
+    SecurityCert,
+    SecurityPrivateKey,
+    SecurityPrivateKeyPassword
```

Comment:
I'm on the edge about this. It's true that NiFi uses SSLContextService and that would make sense that we would use it as well as it is already implemented. But in the kafka extension we currently have the `PublishKafka` and the `ConsumeKafka` processors and as `PublishKafka` uses these attributes for SSL configuration instead of the SSLContextService it would also make sense to have the same properties in `ConsumeKafka` to be consistent within the extension.

Reply:
Not sure what would be the best here. I'm still leaning towards SSLContextService, but we can also add it later and support both approaches. Let's gather more feedback!

Reply:
Added `SSLContextService` to `PublishKafka` and `ConsumeKafka` and added comment for deprecating SSL parameters in `PublishKafka` in 0bf54be561ccab7f0f6da29a9b1e6b8268d5e516. I also removed the `ConsumeKafkaTests.cpp` file as all test cases are now moved to the integration test suite, so there is no need to maintain it anymore.